Acquisition/Poll/source/listener.cpp

```diff
@@ -144,12 +144,16 @@ int main() {
                        size_of_int); // Copy the end packet flag
                 fname[fname_size] = '\0'; // Terminate the filename string
 
-                system("clear");
+                ///@TODO Need to move this to UnixWrappers (or something) to make it DRY appears in monitor.cpp too
+                bool terminalCleared = system("clear") == 0;
```

Comment:
Why create an extra memory assignment? You could just stick the test in the if?

Reply:
Sticking the test in the if might be misleading. 
```c++
if(!system("clear") 
```
Doesn't make sense. An exit code of 0 means success. The if statement reads as if the system didn't clear.